scripts/remote-heal/heal_for_heartbeat.yml

```diff
@@ -0,0 +1,108 @@
+---
+#prybook will try heal for the hear.beat.ping alert
+# * oso-moniotoring pod dead (restarted)
+# * atomic-node service dead（restarted)
+# * host lost connection (try to restart the instance)
+# * if this is a zag pod , delete it from the zabbix
+
+
+- name: check if the this is zag pod
+  hosts: localhost
+
+  pre_tasks:
+    - name: Ensure needed variables are set
+      fail:
+        msg: "Value {{ item }} is not set!"
+      when: "{{ item }} is undefined"
+      with_items:
+      - cli_nodename
+
+    - name: check if the alert is a zag pod
+      set_fact :
+        zag_pod_or_not : true
+      when: cli_nodename | search("oso-rhel7-zagg-web")
+#     when: "'oso-rhel7-zagg-web' in cli_nodename"
+
+    - name: pull the openshift-ansible-private repo for the cred
```

Comment:
rather than having to clone all of private for only the zabbix creds, can we just place the zabbix creds in the opsmedic user's home dir somewhere (with appropriate file permissions)?